### DISCUSSION TITLE
Get a point reflected by paths(scene.compute_paths())

### DISCUSSION THREAD

**Discussion by Chico-Tsunami** [NVlabs/sionna#438] [Q&A]
Hello, I'm making a program in Sionna where I need a/the point(s) reflected after the signal send.

I already get functions paths.sources.numpy() and paths.targets.numpy() but I cant get the points(coordinates) in the middle of the path. 
Can someone help in this?

**Comment by faycalaa** [comment]
Duplicate of #329.
